plugin/hover.py

```diff
@@ -420,3 +425,20 @@ def _update_views_async(self, enable: bool) -> None:
                     session_view.view.settings().set(SHOW_DEFINITIONS_KEY, False)
                 else:
                     session_view.reset_show_definitions()
+
+
+def get_copy_text_from_hover_contents(content: MarkupContent | MarkedString | list[MarkedString]) -> str:
+    if isinstance(content, str):
+        return content
+    elif isinstance(content, dict):
+        return content.get('value', '')
+    elif isinstance(content, list):
+        return " ".join(content)   # pyright: ignore[reportCallIssue, reportUnknownVariableType, reportArgumentType]
+
+
+class LspCopyTextCommand(sublime_plugin.TextCommand):
```

Comment:
Would using the built-in `copy` command be enough? Not sure we need the copied text to show in the status field.

Reply:
It would, do you know if the copy command accepts text as an arg?

I tried to find it but no success.